src/components/FormToggle.vue

```diff
@@ -78,6 +87,14 @@ export default {
             type: Number,
             default: 3,
         },
+        falseValue: {
+            type: [String, Number, Boolean],
```

Comment:
just string, as I said input value could be only string (text)
https://www.w3schools.com/tags/att_input_value.asp

also default value set to `'0'` (string) not `0` (number)

[number/boolean will be converted to string in input value but let be precise and use just strings]